references/sdk.md

```diff
@@ -263,11 +318,21 @@ Get testtuple by key.
 Client.get_traintuple(self, traintuple_key)
 ```
 Get traintuple by key.
+## get_composite_traintuple
+```python
+Client.get_composite_traintuple(self, traintuple_key)
```

Comment:
It should be `composite_traintuple_key` instead of `traintuple_key` (similar to `composite_algo_key` in `get_composite_algo`) Otherwise it may introduce some confusion.

Reply:
Yes, it seems that I forgot to replace it indeed